server/src/domain/asset/response-dto/asset-response.dto.ts

```diff
@@ -31,6 +31,7 @@ export class AssetResponseDto {
   people?: PersonResponseDto[];
   /**base64 encoded sha1 hash */
   checksum!: string;
+  projectionType!: ProjectionType;
```

Comment:
We can remove this field. If you add `projectionType` to `ExifResponseDto`, we will be able to get its value from the `exifInfo` field